src/Pickles/Pickles.BaseDhtmlFiles/Index.html

```diff
@@ -14,7 +14,7 @@
     <!--[if lt IE 9]>
       <script src="js/html5.js"></script>
     <![endif]-->
-    #### EMBED EXPERIMENTALS ####
```

Comment:
Hey @dirkrombauts , I removed this accidentally when copying back `Index.html` after some changes.  I've put it back in now, with a more HTML-friendly syntax and updated the cs file that uses it as well.